### ISSUE TITLE
Question about training settings for FG2

### ISSUE THREAD

**Issue by Mabel0403** [vita-epfl/FG2#5]
Hi Zimin,
Thank you for sharing your work! I'm currently training a model using your FG2 structure, but the performance is not as good as expected. I was wondering if you could share the training details you used, such as the number of **epochs**, **learning rate**, and **optimizer**. Are these settings the same as those used in your CCVPE paper (epoch=15, learning rate=1e-4, optimizer=AdamW)?

Thanks in advance!

**Comment by syt06007** [comment]
I have the same problem. Have you found a solution since then?

**Comment by Mabel0403** [comment]
Unfortunately, I’m still trying to figure it out.

**Comment by ZiminXia** [comment]
I just pushed the training code. Hope it resolves your questions.
Thanks.
